app/javascript/widget/components/PreChat/Form.vue

```diff
@@ -5,8 +5,8 @@ import { mapGetters } from 'vuex';
 import { getContrastingTextColor } from '@chatwoot/utils';
 import { isEmptyObject } from 'widget/helpers/utils';
 import { getRegexp } from 'shared/helpers/Validators';
-import { useMessageFormatter } from 'shared/composables/useMessageFormatter';
-import routerMixin from 'widget/mixins/routerMixin';
+import messageFormatterMixin from 'shared/mixins/messageFormatterMixin';
```

Comment:
@fayazara The `messageFormatterMixin` is already removed, right?

Reply:
@iamsivin fixed this, PTAL